client/src/components/Sidebar/Sidebar.scss

```diff
@@ -87,6 +87,14 @@
   &--mobile &__collapse-toggle {
     display: none;
   }
+
+  .icon--menuitem {
+    width: 1.25em;
+    height: 1.25em;
+    min-width: 1.25em;
+    margin-inline-end: 0.5em;
+    vertical-align: text-top;
+  }
```

Comment:
This was also previously defined in `_main-nav.scss`, and duplicated in `MainMenu.scss` but only for the main menu icons.